src/vs/workbench/browser/parts/editor2/nextEditorPart.ts

```diff
@@ -350,7 +350,7 @@ export class NextEditorPart extends Part implements INextEditorGroupsService, IN
 		}
 	}
 
-	removeGroup(group: INextEditorGroupView | GroupIdentifier): void {
+	removeGroup(group: INextEditorGroupView | GroupIdentifier = this.activeGroup): void {
```

Comment:
This will unfortunately not work if the group where the editor gets closed is not active. This happens when you just click on the tab close button while the group is not active. I suggest to expose the group parameter to the accessor and then using it.